packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -216,6 +222,17 @@ contract SystemConfig_Setters_Test is SystemConfig_Init {
         assertEq(systemConfig.scalar(), newScalar);
     }
 
+    /// @dev Tests that `setGasConfigEcotone` updates the gas limit successfully.
+    function testFuzz_setGasConfigEcotone_succeeds(uint32 basefeeScalar, uint32 blobBasefeeScalar) external {
+        vm.expectEmit(true, true, true, true);
```

Comment:
nit: use `vm.expectEmit(address)` - it assumes the 4 bools are set to true and ensures the event comes from the correct contract